Release 0.91 doc update (#2407)
RedisLabs/redislabs-docs: content/rdi/quickstart/write-behind-guide.md

- The `keys` section specifies the field(s) in the table that are the unique constraints in that table.

- The `mapping` section is optional and used to map a subset of the fields in the Redis Hash or JSON document to the columns that will be written to the target table.

### Apply Filters & Transformations to Write Behind

The RDI Write Behind jobs can apply filters and transformations to the data before it is written to the target. Specify the filters and transformations under the `transform` section.

#### Filters

Use filters to skip some of the data and not apply it to target.
Filters can apply simple or complex expressions that take as arguments, the Redis entry key, fields and even the change op code (create, delete, update, etc.).
Read [here](reference/data-transformation-block-types/filter.md) for more information about filters.

#### Transformations

Transformations manipulate the data in one of the following ways:

- Renaming a field
- Adding a field
- Removing a field
- Mapping source fields to use in output

To learn more about transformations, read [here](data-transformation/data-transformation-pipeline.md).

## How to Provide Target's Secrets

> Target's secrets (e.g. TLS certificate) can be read from a path on the Redis Nodes file system. This allows to consume secrets injected from secret stores.

Reviewer: It felt like there was a missing word in the second sentence. I just left a possible alternative suggestion to consider. Also, I wonder if "on the Redis Nodes file system" should be changed slightly to something like: "on the Redis node's file system."
```suggestion
The target's secrets (such as TLS certificates) can be read from a path on the Redis Nodes file system. This allows the consumption of secrets injected from secret stores.
```

## Deploying the Write Behind Pipeline

> Starting the pipeline is done by running the [`deploy`](reference/cli/redis-di-deploy.md) command:

Reviewer: ```suggestion
To start the pipeline, run the [`deploy`]({{<relref "/rdi/reference/cli/redis-di-deploy">}}) command:
```

```bash
redis-di deploy
```

> You can check that the pipeline is running, receiving and writing data, using the [`status`](reference/cli/redis-di-status.md) command:

Reviewer: ```suggestion
You can check that the pipeline is running, receiving, and writing data, using the [`status`]({{<relref "/rdi/reference/cli/redis-di-status">}}) command:
```

| Deleted incoming events by stream   | `rdi_metrics_incoming_entries{data_source:"…",operation="deleted"}`                                 |
| Filtered incoming events by stream  | `rdi_metrics_incoming_entries{data_source:"…",operation="filtered"}`                                |
| Malformed incoming events by stream | `rdi_metrics_incoming_entries{data_source:"…",operation="rejected"}`                                |
| Total events per stream (snapshot)  | `rdi_metrics_stream_size{data_source:""}`                                                           |
| Time in stream (snapshot)           | `rdi_metrics_stream_last_latency_ms{data_source:"…"}`                                               |

To use the metrics you can either:

- Run the [`status`](reference/cli/redis-di-status.md) command:

  ```bash
  redis-di status
  ```

- Scrape the metrics using RDI Prometheus exporter

## Upgrading

If you need to upgrade RDI, you should use the [`upgrade`](reference/cli/redis-di-upgrade.md) command that provides zero downtime upgrade:

```bash
redis-di upgrade ...
```

> Read [here](upgrade/upgrade-guide.md) about upgrade.

Reviewer: ```suggestion
See the [upgrade guide]({{<relref "/rdi/upgrade">}}) for more information.
```
